DESIGN.md

```diff
@@ -107,6 +107,21 @@ Here you find [i18next](https://www.i18next.com/) locale files (JSON format).
 
 Note that for adding a new locale you must also add an entry in the whitelist in `app/helpers/i18n_helper.js` (line 42).
 
+
+<a name='layout'></a>
+## App layout
```

Comment:
Would it make sense to include a picture? I think you once had some pictures in a discussion thread, right?

Reply:
Yes, including picture make sense